jdaviz/configs/mosviz/helper.py

```diff
@@ -254,6 +254,8 @@ def _is_world_flipped(self):
     def _row_click_message_handler(self, msg):
         self._handle_image_zoom(msg)
         self._handle_flipped_data()
+        # expose the row to vue for each of the viewers
+        self.app.state.settings = {**self.app.state.settings, 'mosviz_row': msg.selected_index}
```

Comment:
would be nice to store this under something other than "settings", but creating a top-level traitlet for this in the app is too config-specific.  Open to other suggestions - but it must be a traitlet that we can pass through into `container.vue`.

Reply:
I have no better suggestion. Let's do this for now and see.